vite.config.ts

```diff
@@ -10,7 +10,8 @@ export default defineConfig({
     alias: {
       '@': path.resolve(__dirname, './src'),
       'styles': path.resolve(__dirname, './src/styles'),
-      'typings': path.resolve(__dirname, './src/typings')
+      'typings': path.resolve(__dirname, './src/typings'),
+      'utils': path.resolve(__dirname, './src/utils'),
```

Comment:
Same point as above, the more aliases we provide the more config we have and needs parsed, all just to save us typing  `@/`.